src/cedar.js

```diff
@@ -552,6 +560,68 @@ export default class Cedar {
   }
 
   /**
+   * CREATE ACCESIBLE TABLE
+   *
+   * Create an accessible invisible table
+   * TODO:
+   *  - Head
+   *  - body
+   *  - invisible
+   *  - accessible order....
+   */
+
+  _accessibleCharts (data, series) {
+    // Grab element that the user passed in
+    const parentEl = document.getElementById(this._elementId.substring(1));
+    // Grab the vega created element
+    const vegaEl = parentEl.firstChild;
+    // Create table...
+    const tableEl = document.createElement('table');
+    const tableHead = this._tableHead(series);
+    const tableBody = this._tableBody(data, series);
+    // Put everything together
+    tableEl.appendChild(tableHead);
+    tableEl.appendChild(tableBody);
+    // Make table invisible
+    tableEl.setAttribute('style', 'display: none');
+    // Assign table
+    parentEl.insertBefore(tableEl, vegaEl);
```

Comment:
Yeah, can add an aria-role to the `vegaEl` to get the screen readers to skip it.